src/main/java/ui/components/pickers/PickerMilestone.java

```diff
@@ -64,6 +62,60 @@ public Node getNewlyAssignedMilestoneNode() {
         return milestone;
     }
 
+    public Node getDetailedMilestoneNode() {
+        HBox matchingMilestone = new HBox();
+        matchingMilestone.setSpacing(3);
+        matchingMilestone.setPadding(new Insets(3, 3, 3, 3));
+        matchingMilestone.setStyle("-fx-border-width: 0 0 1 0; -fx-border-style: solid;");
+
+        HBox milestoneNodeBox = createMilestoneNodeBox();
+        Label separator = new Label("|");
+        HBox milestoneDetailsBox = createMilestoneDetailsBox();
+
+        matchingMilestone.getChildren().setAll(milestoneNodeBox, separator, milestoneDetailsBox);
+        return matchingMilestone;
+    }
+
+    private double calculateProgress(TurboMilestone milestone) {
```

Comment:
Should only pass the required parameters here instead of the whole milestone object.